massgov/openmass
# DP-18837-fix-binder-navigation

Description:
Reverts massgov/openmass#316
Revert "RelatedNodes - several simple queries instead of 1 complex query"

Jira: (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-18837

To Test:

Visit /info-details/covid-19-response-reporting and you should see This is a part of...." at the top. Table of contents will work.  Previous and next buttons at the bottom work correctly.
